install-panel.sh

```diff
@@ -867,6 +882,40 @@ configure_nginx() {
   echo "* nginx configured!"
 }
 
+build_panel() {
+  echo "* Building your panel..."
+
+  # Check if nodejs is already installed
+  if node -v &>/dev/null; then
+      echo "* The dependencies are already installed, skipping this step..."
+    else
+      case "$OS" in
+        debian | ubuntu)
+          curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - && apt-get install -y nodejs
+        ;;
+        centos)
+          [ "$OS_VER_MAJOR" == "7" ] && curl -sL https://rpm.nodesource.com/setup_14.x | sudo -E bash - && sudo yum install -y nodejs yarn
+          [ "$OS_VER_MAJOR" == "8" ] && curl -sL https://rpm.nodesource.com/setup_14.x | sudo -E bash - && sudo dnf install -y nodejs
+        ;;
+      esac
+    fi
+    
+    # Install yarn and build the panel
+    print_warning "This process takes a few minutes, please do not cancel it."
+    if [ "$PTERODACTYL_VERSION" == "v2" ]; then
+        npm i -g yarn
+        cd /var/www/pterodactyl
+        yarn install
+        yarn add @emotion/react
+        yarn build
```

Comment:
> Why are you building non production build? `build:production`

Well, v2 is development, I don't know if it could be built as a stable and usable version.